skills/mindroom-docs/references/page__chat-commands__index.md

```
@@ -141,6 +142,21 @@ Schedule type cannot be changed (one-time to recurring or vice versa) -- cancel
 
 **Aliases:** `!editschedule`, `!edit-schedule`
 
+### `!model`
+
+Show or switch the model that every agent, team, and the router uses in the current thread.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)